static/app/views/performance/transactionSummary/transactionReplays/transactionReplays.tsx

```diff
@@ -35,13 +36,7 @@ function TransactionReplays() {
 
   return (
     <PageLayout
-      location={{
-        ...location,
-        query: {
-          ...location.query,
-          statsPeriod: '90d',
-        },
-      }}
```

Comment:
This seems like the root problem from the ticket: we're overriding whatever was there before with 90d.